src/main/java/org/folio/orders/utils/ResourcePathResolver.java

```diff
@@ -96,6 +99,9 @@ private ResourcePathResolver() {
     apis.put(USERS, "/users");
     apis.put(ORDER_SETTINGS, "/orders-storage/settings");
     apis.put(ROUTING_LISTS, "/orders-storage/routing-lists");
+    apis.put(INSTANCE_STORAGE, "/inventory/instances");
+    apis.put(HOLDINGS_STORAGE, "/holdings-storage/holdings");
+    apis.put(ITEMS_STORAGE, "/inventory/items");
```

Comment:
for Inventory I see that we have InventoryUtils.INVENTORY_LOOKUP_ENDPOINTS
Better to have single declaration of api endpoint in the module